test/cbmc/proofs/ARP/ARPProcessPacket/ARPProcessPacket_harness.c

```diff
@@ -47,5 +48,29 @@ void harness()
         pxARPWaitingNetworkBuffer = NULL;
     }
 
-    eARPProcessPacket( &xARPFrame );
+    /*
+     * The assumption made here is that the buffer pointed by pucEthernetBuffer
+     * is at least allocated to sizeof(ARPPacket_t) size but eventually a even larger buffer.
+     * This is not checked inside eARPProcessPacket.
+     */
+    uint8_t ucBUFFER_SIZE;
+
+    __CPROVER_assume( ucBUFFER_SIZE < CBMC_MAX_OBJECT_SIZE );
```

Comment:
Isn't this true automatically? the `ucBUFFER_SIZE` is an unsigned 8-bit integer meaning that it will never go beyond 255. I assume that `CBMC_MAX_OBJECT_SIZE` will at least be 16-bit value...

Reply:
updated